src/GitLinkTask/GitLink.targets

```diff
@@ -1,11 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <UsingTask TaskName="GitLinkTask.LinkProject" AssemblyFile="$(MSBuildThisFileDirectory)..\..\GitLinkTask.dll" />
+  <UsingTask TaskName="GitLink" AssemblyFile="$(MSBuildThisFileDirectory)GitLinkTask.dll" />
 
-    <Target Name="GitLinkProject" AfterTargets="CopyFilesToOutputDirectory">
-        <LinkProject
-            SolutionDirectory="$(SolutionDir)"
-            ProjectName="$(MSBuildProjectName)" />
-    </Target>
+  <Target Name="LinkSymbolsToRemoteGitRepo" AfterTargets="Compile">
```

Comment:
It should be possible to control if `LinkSymbolsToRemoteGitRepo` target is run. Ideally using boolean msbuild property and target condition. This property value should probably also consider `$(DesignTimeBuild)` (i think Compile target is also called for design time builds).

Also please make ContinueOnError configurable.

I could try to create PR to demonstrate what i mean, just not sure where to target it.